Functions/Assertions/Should.ps1

```diff
@@ -53,17 +53,18 @@ function Should {
 
     begin {
         $inputArray = [System.Collections.Generic.List[PSObject]]@()
+        $lineNumber = $MyInvocation.ScriptLineNumber
```

Comment:
Why did those definitions need to move to begin? I don't see them used until `end` where they were defined before.

Reply:
Reverted